extensions/devservices/keycloak/runtime/src/main/java/io/quarkus/devservices/keycloak/runtime/KeycloakDevServicesRecorder.java

```diff
@@ -0,0 +1,106 @@
+package io.quarkus.devservices.keycloak.runtime;
+
+import java.util.function.Supplier;
+
+import org.jboss.logging.Logger;
+import org.keycloak.admin.client.Keycloak;
+import org.keycloak.admin.client.KeycloakBuilder;
+
+import io.quarkus.devconsole.runtime.spi.DevConsolePostHandler;
+import io.quarkus.runtime.annotations.Recorder;
+import io.vertx.core.Handler;
+import io.vertx.core.MultiMap;
+import io.vertx.core.Vertx;
+import io.vertx.core.http.HttpHeaders;
+import io.vertx.ext.web.RoutingContext;
+import io.vertx.mutiny.ext.web.client.WebClient;
+
+@Recorder
+public class KeycloakDevServicesRecorder {
+    private static final Logger LOG = Logger.getLogger(KeycloakDevServicesRecorder.class);
+
+    public Supplier<String> getProviderName() {
```

Comment:
All of this stuff can just be done directly in the processor on the -deployment side.

Reply:
As far as I recall it was not working